tests/compilers/wast.rs

```diff
@@ -21,13 +21,6 @@ use wasmer_wast::Wast;
 // }
 include!(concat!(env!("OUT_DIR"), "/generated_spectests.rs"));
 
-// This prefixer returns the hash of the module to prefix each of
-// the functions in the shared object generated by the `NativeEngine`.
-fn _native_prefixer(bytes: &[u8]) -> String {
-    let hash = blake3::hash(bytes);
-    format!("{}", hash.to_hex())
-}
-
```

Comment:
Can we keep this function, it might be useful for the future

Reply:
Done.